unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/Avatar.cs

```diff
@@ -39,7 +38,7 @@ public class Avatar : IAvatar
 
         internal Avatar(IAvatarCurator avatarCurator, ILoader loader, IAnimator animator,
```

Comment:
What do you think of removing IAnimator from avatar and handling it directly inside IAvatarEmotesController? I think you can move the Prepare() methods into IAvatarEmotesController and remove it completely for the Avatar class. In this manner, all responsibility for handling the IAnimator would be encapsualted in IAvatarEmotesController